worldengine/cli/main.py

```diff
@@ -104,13 +103,11 @@ def generate_plates(seed, world_name, output_dir, width, height,
     filename = '%s/plates_%s.png' % (output_dir, world_name)
     # TODO calculate appropriate sea_level
     sea_level = 1.0
-    draw_simple_elevation_on_file(world.elevation['data'], filename, width,
-                                  height, sea_level)
+    draw_simple_elevation_on_file(world, filename, None)
```

Comment:
I assume you are referring to line 106/105 and if you are then you're right, that line can be removed as it isn't doing anything. I just didn't think of it at the time.

While platec has made some assumptions concerning sea level at this point of production no ocean generation has been done. As such we need to pass a value that tells draw_simple_elevation_on_file not to try and reference the ocean mask.

I could examine the world and see if it has an ocean mask, at which point it will run as it currently runs, but at that point it would be ignoring the sea_level anyway. Since draw_simple_elevation_on_file doesn't pass the sea_level on to anything else (except for draw_simple_elevation) it would result in a change of code that doesn't really change anything.